packages/react/src/components/searchBar/tests/SearchBar.spec.tsx

```diff
@@ -1,124 +1,106 @@
-import {mount, shallow} from 'enzyme';
-import {keyCode} from '../../../utils/InputUtils';
-import {Svg} from '../../svg/Svg';
+import {render, screen} from '@test-utils';
+import userEvent from '@testing-library/user-event';
+import {shallow} from 'enzyme';
 import {SearchBar} from '../SearchBar';
 import {searchBarPropsScenarios} from './SearchBarPropsScenarios.mock';
 
 describe('SearchBar', () => {
     const requiredProps = {...searchBarPropsScenarios[0]};
-    afterEach(() => {
-        jest.restoreAllMocks();
-    });
 
-    it('should not throw on render', () => {
+    it('does not throw', () => {
         expect(() => {
-            searchBarPropsScenarios.forEach((props) => shallow(<SearchBar {...props} />));
+            searchBarPropsScenarios.forEach((props) => render(<SearchBar {...props} />));
         }).not.toThrow();
     });
 
-    it('should have a container div with the search-bar class and without the search-bar-loading and search-bar-disabled classes by default', () => {
-        const containerDiv = shallow(<SearchBar {...requiredProps} />)
-            .find('div')
-            .first();
+    it('render the search bar', () => {
+        render(<SearchBar {...requiredProps} />);
 
-        expect(containerDiv.hasClass('search-bar')).toBe(true);
-        expect(containerDiv.hasClass('search-bar-loading')).toBe(false);
-        expect(containerDiv.hasClass('search-bar-disabled')).toBe(false);
+        expect(screen.getByRole('textbox')).toBeInTheDocument();
+        expect(screen.getByRole('img', {name: 'search icon'})).toBeInTheDocument();
     });
 
-    it('should have a container div with search-bar-loading class when searching is passed as prop', () => {
-        const containerDiv = shallow(<SearchBar {...requiredProps} searching />)
-            .find('div')
-            .first();
+    it('renders a loading animation when is searching', () => {
+        const {container} = render(<SearchBar {...requiredProps} searching />);
 
-        expect(containerDiv.hasClass('search-bar-loading')).toBe(true);
+        // eslint-disable-next-line testing-library/no-container
+        expect(container.querySelector('.loading-spinner')).toBeInTheDocument();
```

Comment:
Here you could find by role with the new role instead

Reply:
:derp: